H/htslib/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "htslib"
+version = v"1.10.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/samtools/htslib/releases/download/$(version)/htslib-$(version).tar.bz2",
+                  "e3b543de2f71723830a1e0472cf5489ec27d0fbeb46b1103e14a11b7177d1939")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/htslib-*
+./configure \
+    CFLAGS=-I${prefix}/include CPPFLAGS=-I${prefix}/include LDFLAGS=-L${prefix}/lib \
+    --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+
+# On Windows, product files are renamed so that BB can find them.
+if [[ ${target} == *-mingw32 ]]; then
+    # Rename a versioned DLL file.
+    mv ${prefix}/bin/hts-*.dll ${prefix}/bin/libhts.dll
+    # Add .exe extension to executables.
+    for exefile in bgzip tabix htsfile; do
+        mv ${prefix}/bin/${exefile} ${prefix}/bin/${exefile}.exe
+    done
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# NOTE: Configuring i686-w64-mingw32 fails due to 'unable to find the recv()
+# function' error. So we skip it for now.
```

Comment:
This is weird, `./configure` is testing the function by linking to `ws2_32`, and this library has the symbol:
```
sandbox:${WORKSPACE}/srcdir/htslib-1.10.2 # nm /opt/i686-w64-mingw32/i686-w64-mingw32/sys-root/lib/libws2_32.a | grep recv
00000000 I __imp__recvfrom@24
00000000 T _recvfrom@24
00000000 I __imp__recv@16
00000000 T _recv@16
```

Reply:
Yes, it does. But I don't have enough time to debug it further, and it has lower priority because I believe most practitioners will never use 32-bit Windows for sequence analysis of Biology.